.changes/next-release/enhancement-eks updatekubeconfig-80391.json

```diff
@@ -0,0 +1,5 @@
+{
+  "category": "eks update-kubeconfig",
```

Comment:
Same thing here. We should remove this entry since the update already made it to v2.